src/styles/watch.scss

```diff
@@ -0,0 +1,56 @@
+@import './mixin';
+@import './var';
```

Comment:
no need to import it here
just import `./watch` after these files in the `main.scss`